DQM/BeamMonitor/test/Online_BeamMonitor_file.py

```diff
@@ -40,17 +43,21 @@
 )
 #process.MessageLogger.cout.enableStatistics = cms.untracked.bool(True)
 process.source = cms.Source("EmptySource")
-process.source.numberEventsInRun=cms.untracked.uint32(20)
-process.source.firstRun = cms.untracked.uint32(336055)
-process.source.firstLuminosityBlock = cms.untracked.uint32(49)
+process.source.numberEventsInRun=cms.untracked.uint32(100)
+process.source.firstRun = cms.untracked.uint32(346512)
+process.source.firstLuminosityBlock = cms.untracked.uint32(614)
 process.source.numberEventsInLuminosityBlock = cms.untracked.uint32(1)
 process.maxEvents = cms.untracked.PSet(
             input = cms.untracked.int32(100)
 )
 
 #process.load("DQMServices.Core.DQMEDAnalyzer") 
 process.onlineBeamMonitor = cms.EDProducer("OnlineBeamMonitor",
-MonitorName = cms.untracked.string("onlineBeamMonitor"))
+MonitorName         = cms.untracked.string("onlineBeamMonitor"),
+AppendRunToFileName = cms.untracked.bool(False),
+WriteDIPAscii       = cms.untracked.bool(True),
+DIPFileName         = cms.untracked.string("BeamFitResultsForDIP.txt")
```

Comment:
Do we need to add the same options also in
https://github.com/cms-sw/cmssw/blob/d1be6d93d0d1174f154c6374044146cbf5bc929c/DQM/Integration/python/clients/onlinebeammonitor_dqm_sourceclient-live_cfg.py#L128 ?

Reply:
Actually the easiest way would be to clone the onlinebeammonitor configuration so that we do not have to add the extra parameters, but i agree this file has to be changed